horace_core/sqw/@DnDBase/calculate_qw_bins.m

```diff
@@ -0,0 +1,106 @@
+function qw=calculate_qw_bins(win,optstr)
+% Calculate qh,qk,ql,en for the centres of the bins of an n-dimensional sqw or dnd dataset
```

Comment:
checking if this is applicable to a dnd dataset if it is a method of dndbase

Reply:
this is certainly `dnd` method by design. As I remember, sqw object just delegates this operations to dnd object